nibabel/orientations.py

```diff
@@ -230,20 +230,21 @@ def inv_ornt_aff(ornt, shape):
 
 
 @deprecate_with_version('orientation_affine deprecated. '
-                        'Please use inv_ornt_aff instead'
+                        'Please use inv_ornt_aff instead.',
                         '1.3',
```

Comment:
Seems kind of unfair to change the docs to claim it was deprecated since 1.3, since it only just started saying it at 3.0.

```suggestion
                        '3.0',
```